kimu/core/intersection_tool_line_circle.py

```diff
@@ -1,4 +1,6 @@
 import math
+import decimal
```

Comment:
Not sure if it's convetion to use `decimal.Decimal`, but consider using `from decimal import Decimal`.